### PR TITLE
SWG-3640 -- While publishing api and api have unpublish domain ref th…

### PR DESCRIPTION
**Pull request by divyeshvaja**
SWG-3640 -- While publishing api and api have unpublish domain ref then showing prompt and also added force switch for publish api.

### CODE REVIEW

File: src/requests/spec.js [hunk 1]
```diff
@@ -23,15 +23,16 @@ const getSpec = (specType, pathParams, queryParams, accept) => {
...
-    })
+const putSpec = (specType, pathParams, body, queryParams) => {
+  const { SWAGGERHUB_URL, SWAGGERHUB_API_KEY } = config.getConfig()
+  return http({
+    url: [SWAGGERHUB_URL, specType, ...pathParams],
+    auth: SWAGGERHUB_API_KEY,
+    contentType: 'json',
+    method: 'PUT',
+    body,
+    query: queryParams
```
Comment (barryadk) [inline src/requests/spec.js:34]: Could you put the query param above the body?

File: src/support/command/update-command.js [hunk 2]
```diff
@@ -1,29 +1,48 @@
 const BaseCommand = require('./base-command')
 const { putSpec } = require('../../requests/spec')
+const {
```
Comment (barryadk) [inline src/support/command/update-command.js:3]: Could you use the same format as the other - on a single line with no trailing comma?

**Review by barryadk — COMMENTED** [review]
Just need to resolve the Node.js 12 issues 👍

File: src/requests/spec.js (hunk 1 above)
Comment (ben-smartbear) [inline src/requests/spec.js:34]: change the fourth param to `query` so you can do es6 assignment and not bother with `query: queryParams`

Comment (vishalnimase1) [reply src/requests/spec.js:34]: sure. will move param above the body and as @ben-smartbear  suggested will remove rename parameter to query and use es6 assignment.

Comment (divyeshvaja) [reply src/requests/spec.js]: done.

Comment (divyeshvaja) [reply src/requests/spec.js]: done

File: src/support/command/update-command.js [hunk 3]
```diff
@@ -1,29 +1,46 @@
...
     const successMessage = isPublished ? 'published' : 'unpublished'
 
     await this.executeHttp({
-        execute: () => putSpec(type, pathParams, body), 
-        onResolve: this.setSuccessMessage(successMessage)({ 
-          type: type === 'apis' ? 'API' : 'domain',
-          path: `${owner}/${name}/${version}`
-        }),
-        options: { resolveStatus: [403] }
+      execute: () => flags.force ?  putSpec(type, pathParams, body, flags) : putSpec(type, pathParams, body),
```
Comment (CalemRoelofsSB) [inline src/support/command/update-command.js:13]:
```suggestion
      execute: () => flags.force ? putSpec(type, pathParams, body, flags) : putSpec(type, pathParams, body),
```

Comment (CalemRoelofsSB) [reply src/support/command/update-command.js:13]: No need for double space here

**Review by CalemRoelofsSB — APPROVED** [review]
LGTM 👍

File: src/support/command/update-command.js (hunk 3 above)
Comment (divyeshvaja) [reply src/support/command/update-command.js]: removed double space.

**Review by CalemRoelofsSB — APPROVED** [review]

**Review by cka121 — APPROVED** [review]

**Review by barryadk — APPROVED** [review]

**Pull request merged**